src/index.html

```diff
@@ -9,9 +9,165 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action=""
+      method="POST"
+      onsubmit="onSubmit()"
+      autocomplete="off"
+      class="form_style"
+    >
+
+      <fieldset name="Personal information" class="form_field first_fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="fieldset_field first_label">
+        Surname:
+          <input
+            type="text"
+            id="surname"
+            required
+          >
+        </label>
+
+        <label for="name" class="fieldset_field">
+        Name:
+          <input
+            type="text"
+            id="name"
+            required
+          >
+        </label>
+
+
+        <label for="age" class="fieldset_field">
+        How old are You?
+          <input
+            type="number"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label for="birthday" class="fieldset_field">
+        Full date of birth:
+          <input
+            type="date"
+            id="birthday"
+          >
+        </label>
+
+        <label for="terms" class="fieldset_field">
+        I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="terms"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset name="Registration" class="form_field">
+        <legend>Registration</legend>
+        <label for="e-mail" class="fieldset_field first_label">
+      E-mail:
```

Comment:
Текст не може виходити поза батьківський контейнер
```suggestion
        <label for="e-mail" class="fieldset_field first_label">
          E-mail:
```